yarn.lock

```diff
@@ -123,6 +146,14 @@
     tslib "^2.0.0"
     xml2js "^0.4.19"
 
+"@azure/core-xml@^1.0.0":
+  version "1.0.0"
+  resolved "https://registry.yarnpkg.com/@azure/core-xml/-/core-xml-1.0.0.tgz#294b8a8d5ae82bec5699dd8b9847722777e6986c"
+  integrity sha512-VSM3wQomzRZk4SnDK5vKaROON/d3hgfl+D/pfLjpGR8gxRGuO0I8R+Rp/qj6Cq3x7HPgUqrii3/s/RRwDWWdvQ==
+  dependencies:
+    tslib "^2.2.0"
+    xml2js "^0.4.19"
+
 "@azure/data-tables@12.0.0-beta.3":
```

Comment:
should the second `@azure/data-tables` entry be deduplicated ?

Reply:
Try running `npx yarn-deduplicate --scopes @azure`